docs/en/concept/config.md

```diff
@@ -64,6 +64,18 @@ sink {
 }
 ```
 
+#### multi-line support
+
+In `hocon`, multiline strings are supported, which allows you to include extended passages of text without worrying about newline characters or special formatting. This is achieved by enclosing the text within triple quotes **`"""`** . For example:
+
+```
+var = """
+Apache SeaTunnel is a
+next-generation high-performance,
+distributed, massive data integration tool.
+"""
+```
```

Comment:
How about add test case too?
```
sql = """ select * from "table" """
```

Reply:
Sure, multiline text is indeed common in long SQL queries. I will add this case.